src/utils/viewer/viewer.cpp

```diff
@@ -627,6 +627,26 @@ void Viewer::keyPressEvent(KeyEvent& event) {
       controls_(*rgbSensorNode_, "lookDown", lookSensitivity, false);
       agentMoved = true;
       break;
+    case KeyEvent::Key::Seven: {  // test getting handles
+      std::vector<std::string> loadedHandles =
+          resourceManager_.getTemplateHandlesBySubstring();
+      std::stringstream strdat("");
+      for (auto s : loadedHandles) {
+        strdat << s << " | ";
+      }
+
+      LOG(INFO) << "Loaded Object Templates : " << strdat.str();
+      loadedHandles = resourceManager_.getTemplateHandlesBySubstring("chee");
+      strdat.str("");
+      strdat.clear();
+      for (auto s : loadedHandles) {
+        strdat << s << " | ";
+      }
+
+      LOG(INFO) << "Loaded Object Templates matching 'chee' : " << strdat.str();
+
+      break;
+    }
```

Comment:
Good for testing, but seems bulky. Is this necessary for viewer demo? Can we move to SimTest.cpp?

Reply:
Just a debugging/development tool.  I can get rid of it entirely.